apps/meteor/client/views/room/composer/ComposerMessage.tsx

```diff
@@ -85,11 +86,31 @@ const ComposerMessage = ({ tmid, onSend, ...props }: ComposerMessageProps): Reac
 		useCallback(() => LegacyRoomManager.getOpenedRoomByRid(room._id)?.streamActive ?? false, [room._id]),
 	);
 
-	if (!publicationReady) {
+	if (!publicationReady && hasPreviewPublicRoomPermission) {
 		return <ComposerSkeleton />;
 	}
 
-	return <MessageBox key={room._id} tmid={tmid} {...composerProps} showFormattingTips={true} {...props} />;
+	const subscribe = () => {
+		if (!hasPreviewPublicRoomPermission) {
+			// Get room subscription
+			LegacyRoomManager.open({ rid: room._id, typeName: room.t });
+			LegacyRoomManager.computation.invalidate();
+		}
+	};
```

Comment:
`useOpenRoom` already does `LegacyRoomManager.open`. Also, why is this fn called subscribe?

Reply:
Yes, but a lot of other things too. And it does not do the `computation.invalidate` too...